frontend/packages/dev-console/src/components/guided-tour/GuidedTourIntroBanner.tsx

```diff
@@ -0,0 +1,24 @@
bUkzQsmTugtViYAAAAAwPBAcQ0ALvVnHpDUWaztvIaE6u67h/IaORtkaS3jmTuKkwgAAAAAMJxQXAOAQ8GkiQtldJokW6w9vXENlNfIyWBLaxn9xYbRj4uTCgAAAAAwnFBcA4BjQTJ5p4ymqtjldYGPDfGaxhducayXadyycGtvOVa1d/9qw6W19Kox3r5BY6qnYGEAAAAAAMMWxTUAlIAgmbyl6OV1AY8NiR1xqOru+pVq77lN3qTG/G+AtfJ3+YzqHntAo+6dKX/XHfO+/t8nrWtqNnTpq8bz9vYTDe15DwEAAAAAGBGM6wAAgH/IptMny+omFfHrc9TWqo6jjpH687Ne7MivqfqsM9Z4ru+J36j3yqvztgfWZMZuourLvq1gp53/8WQUqvOccxW+9Gp+9lg1aU1pDQAAAAAoAoprACgxTsrr+W3qOPLonIvl2BGHqvrsM9f+Yjaj3vtnqe/6W3LbBP8QN6o87xzF9z9AMmv57RKF6jzvfIUv/jWnbQZ9pjWlNQAAAAAgTyiuAaAElePk9domrde6T2enem+9XZn7HhraRpAqpPi0qao87DApiK3/2hwnr5m0BgAAAAC4QHENACWqnMrrwZbWa+zV1aW+O+9U/533b9xmI1mlp/ip01R5yH9suLBe3RDLa0prAAAAAIArFNcAUMLK4diQ9R4PMhiZfvU9/bT6bvylbPvyoa8zjJktxyp+0vGK77ef5PlDW2Qjjw3heBAAAAAAgEsU1wBQ4kp58nook9brk5k9W333zFT417fytmY5C/bYVZXHHit/2+3ys+AgJ6+ZtAYAAAAAuEZxDQBloBTL63yX1quzS5ao96GHlHnwYdkVPQXZo1SZxOaqOOLrih94gExlVf432EB5TWkNAAAAACgFFNcAUCZKqbwuZGn9z8I576nv0V8r8/hvpD5blD2LzdTXKXbQgYofeIC8cQ2F33Ad5TWlNQAAAACgVFBcA0AZKYUzr3M+0zqXLPPmqu+Z3yr7u+cVfdTqJEO+eBMSCvbaQ/H9viqvIVH8AP905jVnWgMAAAAASgnFNQCUGZeT17FDizdpvSG2s0OZP72kzOz/Vfinl2SXdbmOtF5my7EKvvQFxb64q4LP7SQTj7uO9PfJ62humklrAAAAAEBJobgGgDLkpLzu7BzMNK4zy3fd3XWEdTL1dRr1+GOuY6xdFMr29w/mPG1KawAAAABA0XiuAwAANl6QTN4io6mSinbocymX1siB51NaAwAAAABKDsU1AJSp1crryHUWDGuU1gAAAACAoqO4BoAyNlBenyLKaxQGpTUAAAAAwAmKawAoc5TXKBBKawAAAACAMxTXADAMUF4jzyitAQAAAABOUVwDwDBBeY08obQGAAAAADhHcQ0Aw0iQTN4i6QxJtsBbPSmpo8B7YE2/l/RxgfegtAYAAAAAlASKawAYZoJU8r9kNFWFm7y+Nkgl9zOeN1HSlcYzvQXaByu9YTxzeJBK7mF8/yvGM/MLtA+lNQAAAACgZFBcA8AwVMBjQ64NUslzJMlPNLQHqeRFNtJkSTdLyuZ5r5HuXRkd7ycTn/ETiQckyW8Y946Mt2cBymtKawAAAABASaG4BoBhKt/HhhjPXLOqtF5jn1QiHaSSUxWGnzKeucN4pj8f+41YRn+TtSf4ycT2QTJ5pzFmjQ8f/IZx74RhtI/yd2wIpTUAAAAAoORQXAPAMJbHY0Ou9ROJ89a714Txb/uJxAm2t7/Ryl4hiSJ048yW5x3kJxKfChpTdxhj1jnBXtGYejNPx4ZQWgMAAAAAShLFNQAMc7lOXq9r0nqd+02auDCWSl1uKisnyOgsSXOGsu9IMHA++N2R5302SCV3CxINjxljBvX/KQ+T15TWAAAAAICSRXENACNADpPXG5y0Xhd/s7GdQTJ5XZBKfiKKop208hzsjqGsNdwYz7wlo4tkvFSQSk6pSDS8OpR1cpi8prQGAAAAAJQ04zoAAKB4si3pUyVdr0F8/TeeuWaopfW6hAsW1ths5muyOkHSl5XHD1CX77p7vpbKO1Nfp9FPPD5f0swwjO6oaEy9mc/1++e1bOcZ85ykzQdxOaU1AAAAAKDkUVwDwAiTTadPltWNWn9pfO3GHA8yFGFrW7210f6yOkDSvpLqclmvlItrSW+PaZmz/T/faDGfwrb528hGz9nIjlvPZZTWAAAAAICyQHENACNQNp2eKqvrJQVreflHQSp5STHzhG3zq202u488c56sdhvKGt1XXSWbXef9DIvIRtFb7/5P9NG8OQq1dODJdH26+fpC7zwwef2UpORaXn7ZeN5+lNYAAAAAgHJAcQ0AI1Q23bqLZC+S1ReNZ2LW2r/I6toglXzSWaaW9H9K+p6r/fOkM0glc5oez0W4YOHmNpO5yHjmYEljbWQ/kjRT2ex1wcQJva5yAQAAAACwMSiuAQAlg+IaAAAAAABIebwpFgAAAAAAAAAA+UBxDQAAAAAAAAAoKRTXAAAAAAAAAICSQnENAAAAAAAAACgpFNcAAAAAAAAAgJJCcQ0AAAAAAAAAKCkU1wAAAAAAAACAkkJxDQAAAAAAAAAoKRTXAAAAAAAAAICSQnENAAAAAAAAACgpFNcAgNJhlHEdIVfGM2X/7wAAAAAAgGsU1wCAkmE870PXGXJlrf3AdQYAAAAAAModxTUAoGTYru6nJC1xnSMnkb3HdQQAAAAAAModxTUAoGQEk5tWyNrTJEWuswzRbLui43rXIQAAAAAAKHcU1wCAkhI0pu4znjlQ0nuus2wM45n/MrHYv8e237bfdRYAAAAAAMqdcR0AAIB1CVvbPiHZhJXxXWdZHxuFK2Kp1MuucwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAOTm/wOcN06otww9OAAAAABJRU5ErkJggg=="
```

Comment:
this is a base-64 encoded PNG--can we get a vector version fresh from the brand portal?